app/src/components/GHGIHomePage/Hero.tsx

```diff
@@ -332,9 +347,15 @@ function ProjectTitle({
       color="white"
       data-testid="hero-project-name"
     >
-      {inventory?.city?.project?.name === "cc_project_default"
-        ? t("default-project")
-        : inventory?.city?.project?.name}
+      {inventory
+        ? inventory?.city?.project?.name === "cc_project_default"
+          ? t("default-project")
+          : inventory?.city?.project?.name
+        : city
+          ? city?.project?.name === "cc_project_default"
+            ? t("default-project")
+            : city?.project?.name
+          : t("default-project")}
```

Comment:
Same here:
Could this logic be simplified by creating a variable definition about the return block?
E.g. `const activeCity = inventory?.city ?? city;`
This way you don't need to have all of the ternary operators everywhere here.